### ISSUE TITLE
Message Informaion Interface

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#46]
## User story
1. As a frontend dev
2. I need an interface
3. Which I can use to retrieve data about the messages

## Acceptance criteria
* [ ] Interface that provides all information mentioned in the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) about the messages

(optional) Further propositions of information that could be relevant to the user should be mentioned in comments down below

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by ahaverkamp** [comment]
Found no way to retrieve the cluster per message.
That's why cluster info is currently omitted on the message response.
The schema needs to be defined per topic to be viewed on a message. If it is not defined, I for now return an empty string.
